src/main/scala/org/broadinstitute/dsde/firecloud/model/Profile.scala

```diff
@@ -29,25 +29,20 @@ case class BasicProfile (
     title: String,
     contactEmail: Option[String],
     institute: String,
-    institutionalProgram: String,
     programLocationCity: String,
     programLocationState: String,
     programLocationCountry: String,
-    pi: String,
-    nonProfitStatus: String,
-    termsOfService: Option[String]
+    termsOfService: Option[String],
+    researchArea: Option[String]
```

Comment:
Making this new field optional for the sake of backwards compatibility. Don't want to break anyone's scripts.